core/trino-main/src/main/java/io/trino/security/InjectedConnectorAccessControl.java

```diff
@@ -52,11 +52,18 @@ public InjectedConnectorAccessControl(AccessControl accessControl, SecurityConte
         this.catalogName = requireNonNull(catalogName, "catalogName is null");
     }
 
+    @Override
+    public void checkCanCreateSchema(ConnectorSecurityContext context, String schemaName, Map<String, Object> properties)
+    {
+        checkArgument(context == null, "context must be null");
+        accessControl.checkCanCreateSchema(securityContext, getCatalogSchemaName(schemaName), properties);
+    }
+
     @Override
     public void checkCanCreateSchema(ConnectorSecurityContext context, String schemaName)
     {
         checkArgument(context == null, "context must be null");
-        accessControl.checkCanCreateSchema(securityContext, getCatalogSchemaName(schemaName));
+        accessControl.checkCanCreateSchema(securityContext, getCatalogSchemaName(schemaName), Map.of());
```

Comment:
I was thinking that pass JDK things is safer here than guava that could be from different classloaders

Reply:
In this case - we might need to remove some guava collection. I think we use `ImmutableList`. I think it would be out of scope for this PR. Maybe a follow-up ?

Reply:
I don't there are any issues at the moment. Maybe I will switch to guava even here for coherence.